RecoTracker/TrackProducer/plugins/VertexConstraintProducer.cc

```diff
@@ -40,14 +40,13 @@ Description: <one line class summary>
 // class decleration
 //
 
-class VertexConstraintProducer: public edm::EDProducer {
+class VertexConstraintProducer: public edm::stream::EDProducer<> {
 public:
   explicit VertexConstraintProducer(const edm::ParameterSet&);
-  ~VertexConstraintProducer();
+  ~VertexConstraintProducer() override;
```

Comment:
```suggestion
  ~VertexConstraintProducer() override = default;
```
and remove the empty definition.